### ISSUE TITLE
AG WG Review: SC problematic for closed functionality, new content for level AAA - WCAG2ICT guidance

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#995]
## Summary
Review of proposed WCAG2ICT guidance Level AAA success criteria in the section "Success criteria problematic for closed functionality", as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
* Level A and AA SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
* Additional guidance in SC problematic for closed functionality for Level AAA SCs can be found in the [Level AAA criteria ](https://w3c.github.io/wcag2ict/#level-aaa-criteria) section within [Success criteria problematic for closed functionality](https://w3c.github.io/wcag2ict/#success-criteria-problematic-for-closed-functionality). 
----

## Background information 

Once finalized, this new WCAG2ICT Note will build on the previous version by also covering WCAG 2.2 level AAA success criteria (currently only level A and AA criteria are covered.  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

In order to add level AAA success criteria, we have included a an additional section in Success criteria problematic for closed functionality.

## Content for Review

* [Level AAA criteria ](https://w3c.github.io/wcag2ict/#level-aaa-criteria) section within [Success criteria problematic for closed functionality](https://w3c.github.io/wcag2ict/#success-criteria-problematic-for-closed-functionality).

**Comment by pday1** [comment]
Mary Jo raised on issue #912, moved here for ease of tracking. 

[1.2.6 Sign Language (Prerecorded)](https://w3c.github.io/wcag2ict/#sign-language-prerecorded) - In the description, "translation" should be "interpretation".

**Comment by pday1** [comment]
I found 2 sections where this should be changed in the document. 

The first is NOTE 1 in the main applying SC section
The second is in the SC problematic for closed section.



Current NOTE 1 for 1.2.6 Sign Language (Prerecorded) in main applying SC section: https://w3c.github.io/wcag2ict/#applying-sc-1-2-6-sign-language-prerecorded-to-non-web-documents-and-non-web-software

> NOTE 1 (ADDED)
> To date, meeting this success criteria has proven to be infeasible, as there are not enough human sign language interpreters available to handle a fraction of the volume of video content being produced. As compared to captioning and audio description, sign language interpretation is a very specialized skill. Emerging technologies may, in the future, allow translation from text or speech to sign language directly. At that time, those who need sign language could use such an automated translation tool in the same way people who are blind use a screen reader. This would give people who need to have audio content presented in sign language the same ability to access this content that people who are blind have access to by using their screen readers.
> As always, authors should not rely on such solutions until they are commonly available at a quality accepted by the signing community. In the meantime, providing sign language interpretation continues to be a need for native sign language users, especially in the context of any public service content.


Current description in SC problematic for closed for SC 1.2.6 Sign Language (Prerecorded): https://w3c.github.io/wcag2ict/#problematic-for-closed-sign-language-prerecorded 

> [1.2.6 Sign Language (Prerecorded) (Level AAA)](https://w3c.github.io/wcag2ict/#sign-language-prerecorded)
> Live sign language translation may not currently be logistically feasible for ICT with closed functionality.
> 

Proposed change (change shown in **bold**):

In https://w3c.github.io/wcag2ict/#applying-sc-1-2-6-sign-language-prerecorded-to-non-web-documents-and-non-web-software

NOTE 1 (ADDED)
To date, meeting this success criteria has proven to be infeasible, as there are not enough human sign language interpreters available to handle a fraction of the volume of video content being produced. As compared to captioning and audio description, sign language interpretation is a very specialized skill. Emerging technologies may, in the future, allow **interpretation** from text or speech to sign language directly. At that time, those who need sign language could use such an automated **interpretation** tool in the same way people who are blind use a screen reader. This would give people who need to have audio content presented in sign language the same ability to access this content that people who are blind have access to by using their screen readers.
As always, authors should not rely on such solutions until they are commonly available at a quality accepted by the signing community. In the meantime, providing sign language interpretation continues to be a need for native sign language users, especially in the context of any public service content.


In https://w3c.github.io/wcag2ict/#problematic-for-closed-sign-language-prerecorded

[1.2.6 Sign Language (Prerecorded) (Level AAA)](https://w3c.github.io/wcag2ict/#sign-language-prerecorded)
Live sign language **interpretation** may not currently be logistically feasible for ICT with closed functionality.

**Comment by maryjom** [comment]
Editorial comments on the SC problematic for closed section changes to Level AA guidance:
- 2.1.1 Keyboard (Level AA) - The second sentence is very difficult to parse. Also uses "products" instead of "ICT". Suggest breaking it up into more readable chunks and simplifying the text as follows:
>**When ICT does** not accept input from a keyboard interface**, it is effectively** closed to assistive technologies that **provide** alternate keyboard input methods**. This means that additional** requirements would be needed**, beyond WCAG success criteria,** to ensure that content is operable by people with disabilities **who need** to use alternate keyboard input methods. 
- 3.3.8 Accessible Authentication (Minimum) Not something you changed, but I noticed that "Systems" is used here and in a few of the new Level AAA guidance instead of saying "ICT". I think we had done a previous sweep to use consistent terminology and may have missed this one.

Editorial comments on the SC problematic for closed section Level AAA guidance:
- 1.4.8 - add a comma after the phrase "In non-web software on ICT with closed functionality" and "meeting this" should probably be "satisfying this" and I don't think "success criterion" should be capitalized. There are a few other uses of "meeting this..." is used that should be changed to "satisfying this..." as well.
- 2.2.5 Re-authenticating (Level AAA) - First sentence uses "offer" and suggest instead using "have". Second sentence uses the term "systems" instead of "ICT", so suggest changing the guidance to say (with changes shown in emphasized text):
>ICT with closed functionality may **have** more limitations on how much data can be kept between sessions. This is particularly true of **ICT that is** designed to be used in public environments.
- 2.4.13 appears in the wrong section (in the Level AA) part of SC problematic for closed appendix when should be in [Level AAA criteria](http://w3c.github.io/wcag2ict/#level-aaa-criteria) SC.
- Target size (Enhanced) (Level AAA) also is appearing in the Level AA part of SC problematic for closed appendix when it should be in [Level AAA criteria](http://w3c.github.io/wcag2ict/#level-aaa-criteria).
- 3.1.3 is also a Level AAA appearing in the AA part of SC problematic for closed appendix. I think that's the last one in the wrong place.
- 2.2.5 & 3.3.9 - Similar to 3.3.8, uses "systems that are" and should probably use "ICT that is".
- 3.3.5 - The internal document link to the glossary item isn't working and shows the "[" and "]" markdown. I think it actually needs to be a link to WCAG since WCAG2ICT doesn't have any special interpretation for the term "context-sensitive help".

**Comment by maryjom** [comment]
@pday1 Please re-review the list I provided in the previous comment regarding the SC problematic for closed functionality section. I added a number of editorial comments and have finally finished reviewing this whose section. I am also done reviewing the document.
